<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnExtractorDialogGui</class>
 <widget class="QDialog" name="AnnExtractorDialogGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotation Extractor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="imageLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="2">
       <widget class="QPushButton" name="save_button">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="search_char"/>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="browse_button">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="previous_button">
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="next_button">
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="width_height_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="4">
       <widget class="QLabel" name="preview_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Word:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="counter_label">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Saved count:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLabel" name="out_info_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
